sources for users/sources-dev/readcomiconline/readcomiconline.js

```diff
@@ -621,9 +621,35 @@ module.exports = class Readcomiconline extends Source  {
     searchMangaParse(page /*Int*/, response /*String -- latestUpdatesResponse -- currentPageHtml */, query, filters){
         var page = parseInt(page); //protect WKNodeBrowserify
         var searchMangaSelector = this.searchMangaSelector();
-        var json = [];
         console.log("readcomiconline searchMangaResponse -- ", response);
         var $ = cheerio.load(response);
+        var json = [];
+        var directManga = $('.barTitle',$('.rightBox')).first().text().trim()
+        //checks if the comic was redirected and then makes one page mangapage
+        if (directManga === 'Cover'){
```

Comment:
why is this ===?

Reply:
no reason to use ===, change it. you can do `${directManga}` == 'Cover' as a last resort